### PR TITLE
DS-530/part 2 - border radius tokens

### PR DESCRIPTION
**Pull request by mzyznowska**

### CODE REVIEW

**Comment by zuzannanap** [comment]
left side border radiuses are wrong :<
image

**Comment by zuzannanap** [comment]
In inutNumber borderRadiuses of arrows should be rounded (they weren't, it's not tokens problem)
figma: 
image

**Comment by zuzannanap** [comment]
radiuses in Legend don't work :<

**Comment by zuzannanap** [comment]
SmallToast borders don't work
image

File: packages/design-system/src/components/Toast/index.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const ToastComponents = {
   Container: ({ children, className }: ContainerProps) => {
     return (
       <div
-        className={classNames("rounded-lg shadow-xl empty:hidden", className)}
+        className={classNames("rounded-toast-baseRadius shadow-xl empty:hidden", className)}
```
Comment (zuzannanap) [inline packages/design-system/src/components/Toast/index.tsx:79]: i guess it doesn't work cause it should be 'radiusBase' ;)

**Review by zuzannanap — CHANGES_REQUESTED** [review]

**Comment by mzyznowska** [comment]
> left side border radiuses are wrong :< image

It is correct in my storybook 😅

**Comment by zuzannanap** [comment]
> > left side border radiuses are wrong :< image
> 
> It is correct in my storybook 😅

ok, now it's also ok in my storybook...🙃

**Review by zuzannanap — APPROVED** [review]

**Pull request merged**